src/components/calendar/google_api.tsx

```diff
@@ -6,6 +6,9 @@ const apiKey = process.env.GOOGLE_CAL_API_KEY || "FAKE_KEY";
 const calendarId = process.env.GOOGLE_CAL_ID || "FAKE_ID";
 const googleCalendar = google.calendar({ version: "v3", auth: apiKey });
 
+// For timezone fandangling
+const moment = require('moment-timezone');
```

Comment:
Is it possible to use the `import moment from ...` syntax? If not that's okay.

Reply:
`import moment from ...` tends to work better with type checkers but if `moment-timezone` is just a js library then dw.

Reply:
quick note rather than review: both of these files relate to server-side components so adding extra libs is cheap, later on when u get to working with client-side components you might want to minimise the use of large external packages sent to the user - implementing the calendar natively improved the render time by about 2s.